prometheus_rules/alerts/error-budget.yaml

```
@@ -0,0 +1,21 @@
+groups:
+- name: slo-exporter-slo-error-budget-alerts
+  rules:
+
+  - alert: ErrorBudgetExhausted
+    expr:
+            slo:stable_version{enabled!="false"}
+            * on(slo_version, slo_domain, namespace) group_right()
+            slo:violation_ratio{slo_time_range="4w"}
+            / on (slo_class,slo_domain,slo_version,slo_type,namespace) group_left ()
```

I know that this was simply copied from a different file, but please change this line so that it reads more intuitively.

EDIT: There is more occurrences of this, please change them all.

```suggestion
            / on (slo_version, slo_domain, slo_type, slo_class, namespace) group_left ()
```

---

This is yet to be done in our internal RR/alerts (which used as a base for this files), so let's keep this to a separate (and single purpose) PR.